org.OpenT2T.Sample.SuperPopular.Thermostat/Wink Thermostat/v2/thingTranslator.js

```diff
@@ -0,0 +1,117 @@
+"use strict";
+
+var q = require('q');
+var wh = require("opent2t-translator-helper-wink");
+
+// This code uses ES2015 syntax that requires at least Node.js v4.
+// For Node.js ES2015 support details, reference http://node.green/
+
+const EventEmitter = require("events");
+
+function validateArgumentType(arg, argName, expectedType) {
+    if (typeof arg === 'undefined') {
+        throw new Error('Missing argument: ' + argName + '. ' +
+            'Expected type: ' + expectedType + '.');
+    } else if (typeof arg !== expectedType) {
+        throw new Error('Invalid argument: ' + argName + '. ' +
+            'Expected type: ' + expectedType + ', got: ' + (typeof arg));
+    }
+}
+
+/**
+ * This translator class implements the "InterfaceA" interface.
+ * Since the interface includes a signal (notifiable property), this class
+ * extends from the node's built-in event-emitter class.
+ */
+class WinkThermostat extends EventEmitter {
```

Comment:
>extends EventEmitter [](start = 21, length = 20)

EventEmitter is only needed for translators that emit signals. This one doesn't.